src/app/components/code-block/index.html

```diff
@@ -0,0 +1,56 @@
+    </p>
+    <p>
+      <sky-alert alertType="info">
+        To use the code block component, you must set it up as a dependency in your SKY UX SPA. First,install the package as part of your SPA dependencies: <stache-code>npm install @blackbaud/skyux-lib-code-block --save</stache-code>. Then install its plugin: <stache-code>npm install @blackbaud/skyux-builder-plugin-code-block --save</stache-code>. Next, add <stache-code>SkyCodeBlockModule</stache-code> as both an import and export in your <stache-code>app-extras.module.ts</stache-code> file. Finally, add the code block plugin to your <stache-code>skyuxconfig.json</stache-code> file <stache-code>"plugins": ["@blackbaud/skyux-builder-plugin-code-block"]</stache-code>.
+      </sky-alert>
+    </p>
+  </sky-demo-page-summary>
+
+  <sky-demo-page-properties>
+    <sky-demo-page-property
+      propertyName="code"
+      isOptional="true"
+    >
+      Passes a code sample into the component. This property binds data, so you can use brackets to evaluate the template expression: <stache-code>[code]</stache-code>. Without the brackets, Angular treats the value as a string and initializes the target property with the string value instead of evaluating the data value. This property accepts <stache-code>string</stache-code> values.
+    </sky-demo-page-property>
+    <sky-demo-page-property
+      propertyName="hideCopyToClipboard"
+      isOptional="true"
+      defaultValue="false"
+    >
+      Indicates whether to hide the copy-to-clipboard button in the code-block header. This property accepts <stache-code>boolean</stache-code> values.
+    </sky-demo-page-property>
+    <sky-demo-page-property
+      propertyName="languageType"
+      isOptional="true"
+      defaultValue="markup"
+    >
+      Indicates the language of the code sample. The language type highlights the syntax in your code sample and appears in the header. This property accepts <stache-code>string</stache-code> values.
+    </sky-demo-page-property>
+    <sky-demo-page-property
+      propertyName="hideHeader"
+      isOptional="true"
+      defaultValue="false"
+    >
+      Indicates whether to hide the header portion of the code block. If the <stache-code>languageType</stache-code> is not declared and <stache-code>hideCopyToClipboard</stache-code> is set to true, the header is hidden. This property accepts <stache-code>boolean</stache-code> values.
```

Comment:
I'd change "header portion of the code block" to "code block's header." It just tightens things up a bit and seems less clunky to me.

Does the second sentence mean that users must also not declare `languageType` and must also set `hideCopyToClipboard` to true? If so, then I think we need to rewrite this to emphasize that fact since it's counterintuitive. (Meaning, I'd expect a `hideHeader` property set to true to hide the header no matter what. If that's not the case, then we should make that explicit. i.e. "To hide the header, you must also set `hideCopyToClipboard` to `true` and specify no `languageType`." Or something like that.)

Reply:
Can confirm that the last sentence is not true. I have the `languageType` set to HTML in the code sample, _but_ `hideHeader` is set to false and the code block displays _without_ the header. I think I'm going to remove that sentence altogether.